pvm/ts/tests/bugs.test.ts

```
@@ -26,29 +40,45 @@ describe("Texas Holdem - Data driven", () => {
-            expect(game.smallBlindPosition).toEqual(2);
-            expect(game.bigBlindPosition).toEqual(1);
+        it.skip("should test bug 735 2", () => {
+            game = fromTestJson(test_735_2);
+            // Check the current round
+            const nextToAct = game.getNextPlayerToAct();
+            expect(nextToAct).toBeDefined();
+        });
 
-            const actual = game.getLegalActions(SEAT_2);
-            expect(actual).toBeDefined();
+        it.only("should test bug 735 3", () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)